messaging/broker.go

```diff
@@ -13,23 +12,34 @@ import (
 	"sort"
 	"strconv"
 	"sync"
+	"time"
 
+	"github.com/boltdb/bolt"
 	"github.com/influxdb/influxdb/raft"
 )
 
-// BroadcastTopicID is the topic used to communicate with all replicas.
-const BroadcastTopicID = uint64(0)
+// DefaultPollInterval is the default amount of time a topic reader will wait
+// between checks for new segments or new data on an existing segment. This
+// only occurs when the reader is at the end of all the data.
+const DefaultPollInterval = 100 * time.Millisecond
```

Comment:
What happens if this gets bumped down to something like 10ms? I'm just wondering because it seems like most of the time topic readers will be at the end of their topic. So they'll poll, read whatever came in during that time. Then wait, then poll again.

Just wondering if it's ok to have this be low.